scripts/ci_integration_tests.sh

```diff
@@ -15,7 +15,7 @@ runtimeType=${2,,}
 
 # Check the entered enclave mode and runtime type
 [[ "mock simulation debug release" =~ (^|[[:space:]])$enclaveMode($|[[:space:]]) ]] && echo $enclaveMode || ( echo "Wrong enclave mode entered: $enclaveMode."; exit 1 )
-[[ "graalvm gramine" =~ (^|[[:space:]])$runtimeType($|[[:space:]]) ]] && echo runtimeType || ( echo "Wrong runtime type entered: $runtimeType."; exit 1 )
+[[ "graalvm gramine" =~ (^|[[:space:]])$runtimeType($|[[:space:]]) ]] && echo &runtimeType || ( echo "Wrong runtime type entered: $runtimeType."; exit 1 )
```

Comment:
Why did you added the ampersand to the runtimeType? Were you planing to use the dollar sign instead to print the value of the variable?